### PR TITLE
BPMN-RPA Studio

### PR DESCRIPTION
**Pull request by joostvangils**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23321)

### CODE REVIEW

**Review by ImJoakim — COMMENTED** [review]
image

File: manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+#  Scope: 
+  InstallerUrl: http://1ic.nl/BPMN_RPA/BPMN_RPA_studio.exe
+  InstallerSha256: 603FC56E4CF6602668D6F9C5264A8DB11F2C637F18E6FDA54CA47D2201EFD340
+  InstallerSwitches:
+    Custom: /norestart
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (ImJoakim) [inline manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml:19]: It doesn't look like it needs the `/norestart` switch. Also, it seems like the silent install is `/silent`

```suggestion
    Silent: /Silent
    SilentWithProgress: /Silent
```

Comment (joostvangils) [reply manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml:19]: OK. Do I need to change it and make a new pull request?

Comment (ImJoakim) [reply manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml:19]: You should be able to commit my suggestion.

File: manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Tags:
+- visual
+- Python
+- programming
+- diagrams
+- flows
+- workflow
+- framework
+- windows
+- linux
```
Comment (ItzLevvie) [inline manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.locale.en-US.yaml:30]:
```suggestion
- visual
- python
- programming
- diagrams
- flows
- workflow
- framework
- windows
- linux
```

File: manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Author: Joost W.N.M. van Gils
+PackageName: BPMN-RPA Studio
+PackageUrl: https://1ic.nl
+License: Freeware
+#LicenseUrl: 
+Copyright: Copyright (c) 1IC B.V.
+#CopyrightUrl: 
+ShortDescription: A Windows Desktop application for creating and modifying flows for the open source BPMN-RPA Python library. Flows can be run on both Windows and Linux.
+Description: A Windows Desktop application for creating and modifying flows for the open source BPMN-RPA Python library. Flows can be run on both Windows and Linux.
+Moniker: BPMN-RPA Studio
```
Comment (ItzLevvie) [inline manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.locale.en-US.yaml:20]:
```suggestion
Moniker: bpmn-rpa-studio
```

File: manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+PackageIdentifier: 1ic.BPMN-RPAstudio
+PackageVersion: 21.8.2
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- .flw
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml:13]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by joostvangils — COMMENTED** [review]
Looks good. Let's merge this.

**Review by joostvangils — COMMENTED** [review]
Looking good. Let's merge

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**